doc/onboarding-extras.md

```diff
@@ -25,6 +25,7 @@
 | `src/async-wrap.*` | @nodejs/async_hooks |
 | `src/node_api.*` | @nodejs/n-api |
 | `src/node_crypto.*` | @nodejs/crypto |
+| `src/inspector_*`, `lib/inspector.js` | @nodejs/v8-inspector |
```

Comment:
(Also, there are other places where the alphabetical stuff is messed up, but it's *mostly* alphabetical and perhaps a second good commit would be to make it *all* alphabetical. But let's get this one line added first...)